examples/simple_loop.rs

```diff
@@ -0,0 +1,100 @@
+/*
+ * Simple Loop example with 4D noise
+ * Daily Sketch 2019/09/23 by Alexis Andre (@mactuitui)
+ *
+ */
+
+use nannou::noise::*;
+use nannou::prelude::*;
+
+fn main() {
+    nannou::app(model).update(update).run();
+}
+
+struct Model {
+    noise: Perlin,
+}
+
+fn model(app: &App) -> Model {
+    let _window = app
+        .new_window()
+        .with_dimensions(1024, 1024)
+        .view(view)
+        .event(window_event)
+        .build()
+        .unwrap();
+    let mut noise = Perlin::new();
+    noise = noise.set_seed(1);
+    Model { noise }
+}
+
+fn update(_app: &App, _model: &mut Model, _update: Update) {}
+
+fn window_event(_app: &App, _model: &mut Model, event: WindowEvent) {
```

Comment:
Hi @MacTuitui thanks heaps for this example. Can I suggest that we remove the window event and update methods from this example? Think it would be good to keep these *simple* examples as concise as possible.